bubuku/broker.py

```diff
@@ -24,8 +25,42 @@ def is_running_and_registered(self):
         return self.id_manager.is_registered()
 
     def stop_kafka_process(self):
+        """
+        Stops kafka process (if it is running) and says if this topic is still is a leader on in ISR list for some
+        partitions
+        :return: True, if broker is stopped and is not a leader or a isr.
+        """
         self._terminate_process()
         self._wait_for_zk_absence()
+        return not self._have_leadership()
+
+    def _is_clean_election(self):
```

Comment:
why not to have `_is_unclean_election` method instead? Which will be 1-to-1 with kafka property name?
Because it looks strange that you invert kafka property and then when you call this method - you invert again (`if not self._is_clean_election()`)